pep-0657.rst

```diff
@@ -210,11 +210,14 @@ Offset semantics
 ^^^^^^^^^^^^^^^^
 
 These offsets are propagated by the compiler from the ones stored currently in
-all AST nodes. They are 1-indexed and a value of 0 will mean that the
-information is not available. Although the AST nodes use ``int`` types to store
-these values, ``uint8_t`` types will be used for storage in the new map to
-minimize storage impact. This decision allows offsets to go from 0 to 255,
-while offsets bigger than these values will be treated as missing (value of 0).
+all AST nodes. The output of the public APIs (``co_positions`` and ``PyCode_Addr2Location``)
+that deal with these attributes use 0-indexed offsets (just like the AST nodes) and designates
+``-1`` as the return code for information not available, but the underlying implementation is
```

Comment:
This is also more consistent with the `PyCode_Addr2Line`, since it also returns `-1` on error but for the sake of consistency we used to convert `-1` to `0` on `PyCode_Addr2Location`. Now that cast won't be needed.